nemo/collections/tts/models/magpietts.py

```diff
@@ -1997,7 +2013,7 @@ def get_inference_attention_plots(
 
         return cross_attention_maps, headwise_cross_attention_maps
 
-    def find_eos_frame_index(self, codes) -> Optional[int]:
+    def find_eos_frame_index(self, codes, eos_detection_method) -> Optional[int]:
```

Comment:
Since you changed the default return of this function, can you update the typehint and docstring to more accurately match the updated function?